tests/target/issue-447.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    if cond {
+    }
+    // This shouldn't be dropped
+    else {
+    }
+}
```

Comment:
Should I combine multiple comment positions into a single `if {} else {}` or have a sepeate blocks for each possible position?